docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -153,57 +149,19 @@
       set_fact:
         config_permdir: "{{ platform_path + '/config/' + software_version }}"
 
-    # Check if DM already exists in either helmv2 or helmv3
-    - name: Get list of releases in helmv2
-      command: >-
-        /usr/local/sbin/helmv2-cli -- helm list --output json
-      environment:
-        KUBECONFIG: "/etc/kubernetes/admin.conf"
-      ignore_errors: yes
-      register: helmv2_releases
-
-    # Ansible returns error code 2 to helmv2_releases if helmv2-cli does not exist
-    - name: Check if helmv2-cli is installed
-      set_fact:
-        helmv2_installed: "{{ false if helmv2_releases.rc == 2 else true }}"
-
-    - name: Query for DM release in helmv2
-      shell: |
-        grep -iq '"name":"deployment-manager"' <<< {{ helmv2_releases.stdout_lines }}
-      ignore_errors: yes
-      register: helmv2_dm_exists
-      when: helmv2_installed
-
-    - name: Get list of releases in helmv3
-      command: >-
-        /usr/sbin/helm list --output json
-      environment:
-        KUBECONFIG: "/etc/kubernetes/admin.conf"
-      ignore_errors: yes
-      register: helmv3_releases
-
-    # Ansible returns error code 2 to helmv3_releases if helmv3 does not exist
-    - name: Check if helmv3 is installed
-      set_fact:
-        helmv3_installed: "{{ false if helmv3_releases.rc == 2 else true }}"
-
-    - name: Query for DM release in helmv3
-      shell: |
-        grep -iq '"name":"deployment-manager"' <<< {{ helmv3_releases.stdout_lines }}
-      ignore_errors: yes
-      register: helmv3_dm_exists
-      when: helmv3_installed
-
     - name: Check nfv-vim.dor_complete flag
       stat:
         path: /var/run/.nfv-vim.dor_complete
       register: dor_complete_file
 
+      # Check deployment-manager system app status
+    - name: Check deployment-manager app is status
+      shell: source /etc/platform/openrc; system application-list | grep deployment-manager | awk '{ print $10 }'
```

Comment:
Please use `--nowrap` with `system` commands otherwise it may be prone to unexpected failures.
`source /etc/platform/openrc; system application-list --nowrap | grep deployment-manager | awk '{ print $10 }'`